cifar10/main.py

```diff
@@ -283,8 +309,24 @@ def train_step(engine, batch):
         model.train()
 
         with autocast(enabled=config["with_amp"]):
-            y_pred = model(x)
-            loss = criterion(y_pred, y)
+            r = np.random.rand(1)
+            if use_cutmix and cutmix_beta > 0 and r < cutmix_prob:
+                # from https://github.com/clovaai/CutMix-PyTorch/blob/master/train.py
+                # generate mixed sample
+                lam = np.random.beta(cutmix_beta, cutmix_beta)
```

Comment:
it would be nice to have a helper method for cutmix op.

Reply:
okay for the signature. 
Done